apps/speed.c

```diff
@@ -508,6 +508,14 @@ static double sigs_results[MAX_SIG_NUM][3];  /* keygen, sign, verify */
 #define COND(unused_cond) (run && count < INT_MAX)
 #define COUNT(d) (count)
 
+#define TAG_LEN 16
+
+static unsigned int mode_op; /* AE Mode of operation */
+static unsigned int aead = 0; /* AEAD flag */
+static unsigned char aead_iv[12]; /* For AEAD modes */
```

Comment:
Why the value 12? Why not more or less? A brief comment explaining this choice would be helpful.

Additionally, it would be better to define a constant for this value and use it, instead of using magic numbers directly, similar to `TAG_LEN`. Thanks!

Reply:
Yes, this is a good point - a define would be appropriate. However there is no need to diverge the code in the backported PR from the master branch code, so not in this PR.